### ISSUE TITLE
Some question about this methods

### ISSUE THREAD

**Issue by DoongLi** [ragavsachdeva/The-Change-You-Want-to-See#6]
Hi Ragav, thank you for your great work. I have a few questions to consult you,. 1.When Query Image and Reference Image have changed a lot of light, can this method work? 2.What is the accuracy of the datasets provided in the paper?And whether the accuracy of image alignment will affect the effect？

**Comment by ragavsachdeva** [comment]
Hi!

1. Yes, the intention for the model is to be invariant to photometric changes. We simulate this by applying colour jitters to query and reference images. So, in theory the change in lighting should be ignored. In practice, colour jittering does not emulate real-world lighting changes very well so it's probably best to try it out. You might benefit from finetuning the model on more realistic lighting changes (see Plasma shadow for example in the Kornia library).

2. By accuracy of the datasets, do you mean the accuracy of the change annotations? If so, then COCO-Inpainted, Kubric-Change and Synthtext-Change are very accurate as they entirely synthetic datasets. VIRAT-STD dataset has some noise in the annotations but it is correct for the most part.

3. Regarding the image alignment, yes, if you can register the images apriori, I expect the model to perform better.

Hope this helps!

**Comment by DoongLi** [comment]
Hi, Ragav, I have read this paper recently, and there are still several questions.

**Comment by DoongLi** [comment]
1. Have you tested on the dataset of Street-View Change Detection？Changesim, VL-CMU-CD etc, and I note that the evaluation datasets in your paper introduction which not include real viewpoint change dataset.

**Comment by DoongLi** [comment]
2. I saw that when you generated the simulation dataset, this operation was performed on a black block. How did this generate? and it is better than use a black block？
![image](https://github.com/ragavsachdeva/The-Change-You-Want-to-See/assets/86915071/b08bf982-893f-4d15-8e1d-c86dddea3cb5)

**Comment by ragavsachdeva** [comment]
Hi, apologies for the delayed response. I haven't been monitoring this repo closely.

1. Regarding datasets: No, I didn't. There are a few reasons as to why. First, the nature of the problem formulation. The way we wanted to pose this problem was open-set / class agnostic "object-level" change detector. This formulation requires a symmetric notion of change i.e. if there is a changed region in one image, by definition a changed region must also be in the other image. Most datasets don't have GT annotations that support this formulation. Second, a lot of the datasets were segmentation based whereas we wanted to train a detector (mainly so that the changes can be counted). Third, some of those weren't publicly available at the time ([see this thread](https://github.com/gmayday1997/SceneChangeDet/issues/7)). For real viewpoint change, have a look at the [followup work](https://github.com/ragavsachdeva/CYWS-3D). 

3. This is by setting "padding_mode" to "reflect". See [this link](https://kornia.readthedocs.io/en/stable/augmentation.module.html). As for whether this is better than using black, I don't know. It probably would make very little to no difference.
